payments/pom.xml

```diff
@@ -39,25 +40,31 @@
             <version>${azure.storage.table.version}</version>
         </dependency>
     <!-- End Azure -->
+    	<dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-bean-validators</artifactId>
+            <version>3.0.0</version>
+        </dependency>
+        <dependency>
+            <groupId>io.springfox</groupId>
+            <artifactId>springfox-swagger2</artifactId>
+            <version>3.0.0</version>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter</artifactId>
-            <version>2.6.4</version>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-web</artifactId>
-            <version>2.6.6</version>
```

Comment:
do we want to remove explicit versions?

Reply:
The spring version would be best left explicit. Otherwise we risk losing control if some libraries, in later versions, lose compatibility and we could run into unexpected errors.

In any case, we could talk about it all together and decide how to proceed.

Reply:
For the other libraries I removed the explicit declaration because the versions coincided with those managed by the spring boot version used. So it was a repeat.

The only one I left explicitly stated was "spring-cloud-starter-openfeign" as it is newer than the in spring-boot version.
I wanted to avoid to introduce possible regressions.